config/express.server.config.js

```diff
@@ -17,6 +17,7 @@ module.exports = function() {
     require('../routes/db.server.routes.js')(app);
     require('../routes/forum.server.routes.js')(app);
     require('../routes/user.server.routes.js')(app);
+    require('../routes/comment.server.routes.js')(app);
```

Comment:
```suggestion
```
There is already a route for posting a comment to a post in '../routes/forum.server.routes.js'

Reply:
Would it be better to move all the comment-related URIs from forum.route to comment.route? And leave this one in?

Reply:
I think it's probably better to have it in the forum.route, because a comment can't exist without a post anyways. Makes more sense.